helmfile.d/0400.kube-prometheus.yaml

```diff
@@ -96,20 +96,27 @@ releases:
             group_wait: "30s"
             group_interval: "5m"
             repeat_interval: "12h"
-            receiver: "slack_general"
+            receiver: "general"
             routes:
             - match:
                 alertname: "DeadMansSwitch"
               receiver: "null"
           receivers:
             - name: "null"
-            - name: "slack_general"
+            - name: "general"
```

Comment:
Rename to `slack_general`?

Reply:
This is the one receiver for both - slack and pagerduty